packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/main/java/com/example/alternate_language_test_plugin/AlternateLanguageTestPlugin.java

```diff
@@ -273,6 +278,28 @@ public void reply(Void value) {
         });
   }
 
+  // TODO: (tarrinneal) Update error handling tests to properly recieve and handle errors.
+  // See issue https://github.com/flutter/flutter/issues/118243
+  @Override
+  public void callFlutterThrowError(Result<Object> result) {
+    flutterApi.throwError(
+        new FlutterIntegrationCoreApi.Reply<Object>() {
+          public void reply(Object value) {
+            result.error(new RuntimeException("An error"));
```

Comment:
Since this is wrong (we won't want to unconditionally throw an error because that won't test the Flutter API actually doing what it's supposed to), let's have no code here and just a TODO. Otherwise we could end up in a situation where the tests pass without anything actually working if we forget to change this.